pkgs/applications/networking/mailreaders/mmh/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, ncurses, autoconf, automake, flex }:
+stdenv.mkDerivation rec {
+  pname = "mmh-unstable";
+  version = "2019-09-08";
+  buildInputs = [ ncurses autoconf automake flex ];
+  rev = "431604647f89d5aac7b199a7883e98e56e4ccf9e";
+  src = fetchurl {
+    url = "http://git.marmaro.de/?p=mmh;a=snapshot;h=${rev};sf=tgz";
+    name = "mmh-${rev}.tgz";
+    sha256 = "1q97p4g3f1q2m567i2dbx7mm7ixw3g91ww2rymwj42cxk9iyizhv";
+  };
+  preConfigure = "./autogen.sh";
```

Comment:
Continuing from @veprbl's comment, then you'll probably won't need this as `autoreconfHook` takes care of running this.